DSharpPlus.sln

```diff
@@ -49,6 +49,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{430C
 		.github\PULL_REQUEST_TEMPLATE.md = .github\PULL_REQUEST_TEMPLATE.md
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DSharpPlus.UnitTests", "DSharpPlus.UnitTests\DSharpPlus.UnitTests.csproj", "{7EAB8D4F-F17B-4594-8701-48CFC18E0882}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DSharpPlus.OAuth", "DSharpPlus.OAuth\DSharpPlus.OAuth.csproj", "{1E53074B-1623-410F-A603-192AC7EF496F}"
```

Comment:
Why a separate OAuth client when Rest project exists and it's there precisely for that reason?